main/src/ca/uwaterloo/flix/language/CompilationMessageKind.scala

```diff
@@ -0,0 +1,69 @@
+package ca.uwaterloo.flix.language
+
+import ca.uwaterloo.flix.language.ast.Ast.SyntacticContext
+
+sealed trait CompilationMessageKind
```

Comment:
I think I would prefer the functional-style with one toString in the trait that does the pattern match.

Reply:
Also lets sort everything alphabetically.